<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrequencyResponse</class>
 <widget class="QDialog" name="FrequencyResponse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="B1">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>20</y>
     <width>351</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Единственная частотная точка</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="B2">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>50</y>
     <width>351</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Линейный закон изменения частоты</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="B3">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>80</y>
     <width>351</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Логарифмический закон изменения частоты</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>341</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="line2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>341</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="line3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>341</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="value1">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="value2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>170</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="value3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>210</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
  </widget>
  <widget class="QPushButton" name="Ok">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
